File: computing-grid/grid-client.py
# Grid Client
# No Copyright. Vladislav Aleinik 2020  

import socket
import socketserver
import sys

from enum import Enum

class ServerRequestType(Enum):
	STILL_ALIVE       = 0
	HERE_IS_YOUR_TASK = 1
	DATA_DONE         = 2




if __name__ == "__main__":

	

File: computing-grid/grid-client.cpp
// No Copyright. Vladislav Aleinik 2019

int main()
{
	// ========================================================================
	// Aquire Connection Socket                                                
	// ========================================================================
	
	int conn_sock = socket(AF_INET, SOCK_STREAM, 0);
	if (conn_sock == -1)
	{
		fprintf(stderr, "[COMPUTING-GRID] Unable to get socket\n");
		exit(EXIT_FAILURE);
	}

	return EXIT_SUCCESS;
}